docs/operator-guide/xyz-projection.md

```diff
@@ -0,0 +1,56 @@
+# XYZ and projection math
+
+[NZTM2000Quad](https://github.com/linz/NZTM2000TileMatrixSet) is not supported by many third party tools especially inside the Vector Tile space. Often tools expect tiles to be served as `/{z}/{x}/{y}.mvt` and assume the projection is [WebMercatorQuad]().
```

Comment:
empty link?

Reply:
I spent a bit of time looking for a good WebMercatorQuad link but didnt really find anything, do you know of a one? maybe just the standard Bing maps docs could be useful https://learn.microsoft.com/en-us/bingmaps/articles/bing-maps-tile-system